shell/browser/web_view_guest_delegate.cc

```diff
@@ -40,10 +40,6 @@ void WebViewGuestDelegate::AttachToIframe(
 
   content::WebContents* guest_web_contents = api_web_contents_->web_contents();
 
-  // Force a refresh of the webPreferences so that OverrideWebkitPrefs runs on
-  // the new web contents before the renderer process initializes.
-  guest_web_contents->NotifyPreferencesChanged();
```

Comment:
The statement is true but its better to have the recalc call closest to the point where we expect.

Reply:
Got it, thanks! 👍